src/scss/hpe/_hpe.defaults.scss

```diff
@@ -16,7 +16,7 @@ $brand-status-colors: (
 );
 $brand-link-color: inherit;
 $link-hover-color: #000;
-$brand-link-text-decoration: underline;
+$brand-link-text-decoration: none;
```

Comment:
It's overriding a tags created by Link (in this case, in the search results overlay):
```
<Link to={this._buildHref(searchResult.data)}>
  <Anchor tag="h2" onClick={this._onSearchClose}>
    <strong>{searchResult.title}</strong>
      ...
```

```
<a class="" href="/brandcentral/brand-elements/color-palette" data-reactid=".2.1.1:1.1:1.1.0.1.1:$Color Palette.1:1">
  <h2 data-reactid=".2.1.1:1.1:1.1.0.1.1:$Color Palette.1:1.0">
    <strong data-reactid=".2.1.1:1.1:1.1.0.1.1:$Color Palette.1:1.0.0">Color Palette</strong>
  </h2>
</a>
```